packages/studio-web/src/app/soundswallower.service.ts

```diff
@@ -101,3 +101,23 @@ export class SoundswallowerService {
     });
   }
 }
+
+export function createAlignedXML(
+  xmlText: string,
```

Comment:
So I assume this is the string returned by the `/assemble` endpoint?

Reply:
yes exactly, we parse it, add the alignments, then return the resulting `Document`